src/controleur/AdherentCompteController.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class AdherentCompteController extends Controller
+{
+
+    // Classes PDO utilisées
```

Comment:
J'ajouterai l'adhrentBD ici comme ça on sait que c'est utilisé. En le nommant pareil sans la majuscule pour qu'on sache facilement ce que c'est.
```suggestion
    // Classes PDO utilisées
    private AdherentBD adherentBD;
```